<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RechercheTodos</class>
 <widget class="QWidget" name="RechercheTodos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>631</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Recherche todos</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>585</width>
      <height>124</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="dRealisationLabel">
       <property name="text">
        <string>Date de réalisation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="dRealisationCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="dIntervalleTodoRechercheLabel">
       <property name="text">
        <string>Interavalle date de Date de réalisation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDateEdit" name="dateRealisationEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="dIntervalleTodoCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="labelDebutTodo">
         <property name="text">
          <string>Début</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dDebutTodo">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelFinTodo">
         <property name="text">
          <string>Fin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dFinTodo">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="rechercheFiltreTodoButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="suppressionFiltreTodoButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer filtre</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="todosTableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>630</width>
     <height>384</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Contenu</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date de réalisation</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="erreurRechercheLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>621</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
